app/src/main/res/layout/third_party_dependency_list_fragment.xml

```diff
@@ -25,9 +25,9 @@
       android:id="@+id/third_party_dependency_list_fragment_recycler_view"
       android:layout_width="match_parent"
       android:layout_height="match_parent"
-      android:background="@color/color_def_white"
+      android:background="@color/component_color_third_party_dependency_list_activity_background_color"
       android:clipToPadding="false"
       android:nestedScrollingEnabled="false"
-      app:list="@{viewModel.thirdPartyDependencyItemList}" />
+      app:list="@{viewModel.thirdPartyDependencyItemList}"/>
```

Comment:
Add ` ` (space) at the end before closing tag `app:list="@{viewModel.thirdPartyDependencyItemList}" />`

Reply:
done